Core/Src/alerts.c

```diff
@@ -134,6 +135,50 @@ static bool stackVsSegmentImbalancePresent(Bms_S* bms)
     return false;
 }
 
+static bool chargerOverVoltagePresent(Bms_S* bms)
+{
+    return (bms->chargerConnected) && (bms->chargerData.chargerVoltage > MAX_CHARGE_VOLTAGE_V);
```

Comment:
If we use `MAX_CHARGE_VOLTAGE_V` - shouldn't there be a bit of a buffer? I doubt the reported voltage from the charger is exactly accurate. In addition the charge voltage requested from the charger could potentially be a bit higher than the max pack voltage, due to resistive losses in the harness